### ISSUE TITLE
uploaded images should use Intervention or any alternatives

### ISSUE THREAD

**Issue by cnvo** [anggadarkprince/infogue#3]
Also thinking about the size constraints for the following: Featured Article Image, Avatar Images (2 or 3 sizes), Article Images, Cover Image, etc.

http://image.intervention.io/

**Comment by anggadarkprince** [comment]
yes, I want to implement this one, create thumb img for featured image make loading faster,, and another issues that you have wrote exactly what I want to do,, I hope I can work this project next month,, supposing someone contribute this project I would be happy..
